resources/web/wwi/X3dScene.js

```diff
@@ -105,6 +108,15 @@ export default class X3dScene {
 
     object.delete();
 
+    WbWorld.instance.robots.forEach((robot, i) => {
+      if (robot.id === 'n'+id) {
+        WbWorld.instance.robots.splice(i,1);
+      }
+    });
+
+    if (document.getElementById('robotWindowButton') !== null)
```

Comment:
```suggestion
    if (document.getElementById('robot-window-button') !== null)
```

Reply:
For this I followed the previous implementation of buttons in the toolbar which previously used a single word (`'infoButton`, 'settingsButton', etc.) 
The `_createToolBarButton` function in `Toolbar.js` adds the word _Button_ at the end of the given name as the id.

Let me know if you still want this change.

![toolbarButtons](https://user-images.githubusercontent.com/100125668/158964922-98c25f71-1780-4800-9b47-3eb5ab1a7c1f.png)

Reply:
Yes, as discussed, I believe we should change them all.

Reply:
These as well as all other inconsistencies have been rectified in latest push.